### DISCUSSION TITLE
Can I use boofuzz to fuzz an event-driven server?

### DISCUSSION THREAD

**Discussion by xinyile** [jtpereyda/boofuzz#578] [Q&A] [answered]
Hi,

I'm new to boofuzz and I'm trying to fuzz an event-driven server. I'm wondering if I can choose boofuzz to do that.

Thanks!

**Comment by jtpereyda** [comment, marked as answer]
I'm not sure how a web server being event-driven would impact your fuzzing requirements. So I'm guessing the answer is basically yes. But you may want to do some more research on how exactly your target works and how that will interact with any fuzzing tool you want to use.

If you already know, feel free to share some more details about why the server being event-driven matters for fuzzing purposes.
